tag.go

```diff
@@ -43,8 +51,34 @@ type htmler interface {
 	HTML() template.HTML
 }
 
+func parseTagEmbed(b interface{}, bb *bytes.Buffer) {
```

Comment:
Have this return a `string` instead of taking in a `bytes.Buffer`, it seems a little too "tied to implementation"